### DISCUSSION TITLE
moose installation

### DISCUSSION THREAD

**Discussion by 412120052** [idaholab/moose#22396] [Q&A Getting Started] [answered]
Hai,
while installing moose through ssh, after entering this command 
 [conda create --name moose moose-libmesh moose-tools]
screen showing like this below
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): - failed

so i cant able to activate moose (conda activate moose). kindly help me with this. Thanking You

**Comment by GiudGiud** [comment, marked as answer]
Hello

It could be that you have the old channel for idaholab
can you please try to remove it `conda config --remove  <channel address>`
then follow these instructions again from adding the channel onwards.
`https://mooseframework.inl.gov/getting_started/installation/conda.html`

Guillaume
